scripts/index-demo-page.js

```diff
@@ -71,21 +71,34 @@
     window.history.replaceState({}, 'vhs demo', query);
   };
 
-  var loadState = function() {
-    var params = {get: function(param) {
-      return null;
-    }};
+  window.URLSearchParams = window.URLSearchParams || function(searchString) {
```

Comment:
Is this a polyfill from somewhere, or just crafted?

Reply:
All of it vaguely comes from stackoverflow, but it's not really a copy paste since the stackoverflow answer was wrong and overly complex.